### PR TITLE
Feature: Add QR generator

### PR DESCRIPTION
**Pull request by tanoggy**
Add QR generator tool

### CODE REVIEW

**Comment by tanoggy** [comment]
hi @dweber019 please help me review

File: plugins/toolbox/src/components/Root/tools.tsx [hunk 1]
```diff
@@ -157,6 +158,12 @@ export const defaultTools: Tool[] = [
     category: 'Backstage',
     description: 'Describes catalog entities in YAML',
   },
+  {
+    id: 'qr-generator',
```
Comment (dweber019) [inline plugins/toolbox/src/components/Root/tools.tsx:162]: Would name it like the others like `qr-code-generate`

File: plugins/toolbox/src/components/Root/tools.tsx [hunk 2]
```diff
@@ -157,6 +158,12 @@ export const defaultTools: Tool[] = [
...
+    name: 'QR generator',
```
Comment (dweber019) [inline plugins/toolbox/src/components/Root/tools.tsx:163]: Would name this `QR Code`

File: plugins/toolbox/src/components/Generators/QR.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const sample = faker.internet.url();
+
+
+  return (
+    <DefaultEditor
+      input={input}
+      setInput={setInput}
+      sample={sample}
+      rightContent={
+        <>
```
Comment (dweber019) [inline plugins/toolbox/src/components/Generators/QR.tsx:17]: Can be removed as not required.

File: plugins/toolbox/src/components/Generators/QR.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { DefaultEditor } from '../DefaultEditor';
+import { faker } from '@faker-js/faker';
+import QRCode from "react-qr-code";
+
+export const QR = () => {
```
Comment (dweber019) [inline plugins/toolbox/src/components/Generators/QR.tsx:6]: Could be named `QRCode` but then we have to name the `import QRCode from "react-qr-code";` differently. Would be more aligned with the changes proposed in `tools.tsx`.

**Review by dweber019 — CHANGES_REQUESTED** [review]
Only some minor changes / suggestions.
I think the name should be changed in `tools.tsx`.

File: plugins/toolbox/src/components/Generators/QR.tsx (hunk 3 above)
Comment (tanoggy) [reply plugins/toolbox/src/components/Generators/QR.tsx]: Please describe for me and continue review my code, many thanks

**Comment by tanoggy** [comment]
I fix yarn lock @dweber019 sr my miss take

**Comment by tanoggy** [comment]
@dweber019 I see everything be ok, please progress my pull request

**Review by drodil — APPROVED** [review]
Looks good, thanks for this!

**Pull request merged**